config/html-elements-plugin/index.js

```diff
@@ -0,0 +1,104 @@
+
+function HtmlElementsPlugin(locations) {
+  this.locations = locations;
+}
+
+HtmlElementsPlugin.prototype.apply = function(compiler) {
```

Comment:
yes, I was about to open an issue for it, let's give this a week see everything is fine and go for it